de-lib/src/main/java/org/iplantc/de/client/models/tool/ToolAutoBeanFactory.java

```diff
@@ -33,4 +34,20 @@
 
     AutoBean<SimpleServiceError> simpleServiceError();
 
+    /**
+     * Build a tool with default values
+     *
+     * @return  a tool
+     */
+    default AutoBean<Tool> geTool() {
+        Tool tool = getTool().as();
```

Comment:
After checking, it looks like `ToolAdminDetailsDialog#show` is the only other place this factory's `getTool` method it called from.